src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -220,7 +213,7 @@ private async Task WriteBatchBytesAsync(UnacknowledgedRenderBatch pending)
 
                 Log.BeginUpdateDisplayAsync(_logger, _client.ConnectionId, pending.BatchId, pending.Data.Count);
                 var segment = new ArraySegment<byte>(pending.Data.Buffer, 0, pending.Data.Count);
-                await _client.SendAsync("JS.RenderBatch", Id, pending.BatchId, segment);
+                await _client.SendAsync("JS.RenderBatch", UnusedRendererId, pending.BatchId, segment);
```

Comment:
Just remove it from here and from the JS side and use 0 always in the client if you need to.

Reply:
Not sure what you mean. The JS-side function has an argument in this position. I have to pass something, or deal with the complexity of overloading on the JS side which is worse. Did you have something else in mind?

Reply:
https://github.com/aspnet/AspNetCore/blob/master/src/Components/Web.JS/src/Boot.Server.ts#L78

Remove the parameter from there. Use 0 in the client-side code as a constant.

Reply:
You can get rid of https://github.com/aspnet/AspNetCore/blob/master/src/Components/Web.JS/src/Platform/Circuits/RenderQueue.ts#L22-L31

Reply:
https://github.com/aspnet/AspNetCore/blob/master/src/Components/Web.JS/src/Platform/Circuits/RenderQueue.ts#L57 -> Pass 0 there

Reply:
Great points. I'll remove it that way.